go-controller/pkg/ovn/egressip.go

```diff
@@ -2196,12 +2188,8 @@ type egressIPController struct {
 	nbClient libovsdbclient.Client
 	// watchFactory watching k8s objects
 	watchFactory *factory.WatchFactory
-	// EgressIP Node reachability total timeout configuration
```

Comment:
interesting; I can see now that having that in config is more than enough. thanks for removing the redundancy here.